docs/reference/aggregations/bucket/frequent-items-aggregation.asciidoc

```diff
@@ -0,0 +1,159 @@
+[[search-aggregations-bucket-frequent-items-aggregation]]
+=== Frequent items aggregation
+++++
+<titleabbrev>Frequent items</titleabbrev>
+++++
+
+A bucket aggregation which finds frequent item sets. Finding frequent item sets 
+is a form of association rules mining that identifies items that are often occur 
+together. Items that are frequently purchased together or log events that tend 
+to co-occur are examples of frequent item sets. Finding frequent item sets helps 
+to discover relationships between different data points (items) in a data set 
+and to create association rules based on those relationships.
```

Comment:
> and to create association rules based on those relationships.

"association rules" isn't a concept I'm familiar with and didn't seem to pop up anywhere else in a quick docs search. It might be a good idea to give a quick explanation or link elsewhere for more context.